lambdas/db-migration/src/migrations/20210715171725_provider_add_allowed_redirects.ts

```diff
@@ -0,0 +1,16 @@
+import * as Knex from 'knex';
+
+export const up = async (knex: Knex): Promise<void> =>
+  await knex.schema.table('providers', (table) => {
+    table
+      .jsonb('allowed_redirects')
```

Comment:
Does Knex not allow array types? It seems like we could do a `text[]` and save the trouble of stringifying/decoding but there might be a Knex reason for not doing that.

Reply:
The internet tells me it's possible with something like this https://stackoverflow.com/questions/50118196/how-to-insert-array-data-type-using-knex-and-potsgres

but there might be reasons to not do it that way. Open to pushback.

Reply:
I had absolutely no idea that this existed. I'll look into it. Good find

Reply:
b3a173b7afaf8df2689665f914d2e14bcb0355fd
7c494fe0c4751150f450c933708f734939b7fb2f